### PR TITLE
Camel Quarkus 2.0.0 release announcement

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

**Review by davsclaus — APPROVED** [review]

File: content/blog/2021/06/camel-quarkus-release-2.0.0/index.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+When it comes to developer experience, Quarkus 2.0 introduces continuous testing.
+Quarkus dev mode (a.k.a. `mvn quarkus:dev`) exists since the very beginnings of Quarkus.
+It not only starts your application so that you can test it manually,
+but it also keeps watching for changes in your workspace, recompiling the modified classes as needed
+and reloading the running application.
+Now with continuous testing, your tests are executed as you change your application code.
+You have to press `r` to enable it.
+You may want to watch [this video](https://www.youtube.com/watch?v=rUyiTzbezjw) to learn more about the new continuous testing feature.
+
+Please refer to [Quarkus 2.0.0.Final announcement](https://quarkus.io/blog/quarkus-2-0-0-final-released/) for more details.
```
Comment (aldettinger) [inline content/blog/2021/06/camel-quarkus-release-2.0.0/index.md:31]: Its 404 today, but I guess it should be fine.

File: content/blog/2021/06/camel-quarkus-release-2.0.0/index.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+* [jOOR](/camel-quarkus/latest/reference/extensions/joor.html) (JVM only)
+* [Kamelet Reify](/camel-quarkus/latest/reference/extensions/kamelet-reify.html) (JVM only)
+* [Protobuf Jackson](/camel-quarkus/latest/reference/extensions/jackson-protobuf.html)
+
+Extensions newly supported in native mode:
+
+* [DigitalOcean](/camel-quarkus/latest/reference/extensions/digitalocean.html)
+* [OpenStack](/camel-quarkus/latest/reference/extensions/openstack.html)
+* [XChange](/camel-quarkus/latest/reference/extensions/xchange.html)
+* [XQuery](/camel-quarkus/latest/reference/extensions/xchange.html)
```
Comment (aldettinger) [inline content/blog/2021/06/camel-quarkus-release-2.0.0/index.md:56]: Maybe it's xquery.html ?

File: content/blog/2021/06/camel-quarkus-release-2.0.0/index.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+
+## Test coverage and closing functionality gaps
+
+In this release, we invested a lot of effort into reviewing existing tests and adding new test cases.
+We proceeded methodically, following the main Camel documentation.
+We took care to cover every use case mentioned on the Camel component pages.
+For example for the HTTP component, we went through the use cases mentioned on its component page](/components/latest/http-component.htm),
```
Comment (aldettinger) [inline content/blog/2021/06/camel-quarkus-release-2.0.0/index.md:64]: Should it be http-component.html ?

File: content/blog/2021/06/camel-quarkus-release-2.0.0/index.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+## Breaking changes and migration steps
+
+Please refer to our [2.0.0 Migration guide](/camel-quarkus/latest/migration-guide/2.0.0.html).
+
+## Release notes
+
+Fixed issues:
+
+* [Milestone 2.0.0-M1](https://github.com/apache/camel-quarkus/milestone/13?closed=1)
+* [Milestone 2.0.0-M2](https://github.com/apache/camel-quarkus/milestone/13?closed=1)
```
Comment (aldettinger) [inline content/blog/2021/06/camel-quarkus-release-2.0.0/index.md:87]: miestone/16  for M2 ?

**Review by aldettinger — COMMENTED** [review]
Few typos proposal. Beyond, it' look good to me. Well done @ppalaga :+1:

**Review by aldettinger — APPROVED** [review]

File: content/blog/2021/06/camel-quarkus-release-2.0.0/index.md (hunk 3 above)
Comment (zbendhiba) [reply content/blog/2021/06/camel-quarkus-release-2.0.0/index.md:64]: and there is a missing "[" for this link. Look at the preview

File: content/blog/2021/06/camel-quarkus-release-2.0.0/index.md (hunk 1 above)
Comment (ppalaga) [reply content/blog/2021/06/camel-quarkus-release-2.0.0/index.md:31]: Yeah, it is blocked by this one: https://github.com/quarkusio/quarkusio.github.io/pull/1037

File: content/blog/2021/06/camel-quarkus-release-2.0.0/index.md (hunk 2 above)
Comment (ppalaga) [reply content/blog/2021/06/camel-quarkus-release-2.0.0/index.md:56]: Thanks for noticing, it is saxon.html

File: content/blog/2021/06/camel-quarkus-release-2.0.0/index.md (hunk 3 above)
Comment (ppalaga) [reply content/blog/2021/06/camel-quarkus-release-2.0.0/index.md:64]: +1, thanks!

File: content/blog/2021/06/camel-quarkus-release-2.0.0/index.md (hunk 4 above)
Comment (ppalaga) [reply content/blog/2021/06/camel-quarkus-release-2.0.0/index.md:87]: Yep, thanks, another copy and paste mistake

**Comment by ppalaga** [comment]
3d7f81e:

* Addressed the comments of @aldettinger 
* Added a paragraph about the extent of the test expansion effort https://github.com/apache/camel-website/pull/596/files#diff-ea0bf99904cb0b8bac8dfa648281b0d6b160e7e63cf67fab037cdf640924e617R73

Still blocked by https://github.com/quarkusio/quarkusio.github.io/pull/1037

**Comment by ppalaga** [comment]
ee3e19f: fixed @zbendhiba 's "and there is a missing "[" for this link. Look at the preview"

**Comment by ppalaga** [comment]
OK, the blocking PR was merged and https://quarkus.io/blog/quarkus-2-0-0-final-released/ is available

**Review by zbendhiba — APPROVED** [review]

**Pull request merged**